### PR TITLE
first steps and discovery reworked

### PR DESCRIPTION
**Pull request by ma-zah**

### CODE REVIEW

File: first-steps.mdx [hunk 1]
```diff
@@ -18,25 +18,40 @@ The second page will render a screenshot of the url you provided. It tests that
...
+  />
+</Frame>
+
+If you choose the `log in` test, we'll ask you for **test user credentials** \(test username and a password\).
 The test credentials should fit the URL you provided above.
 
-<Frame caption="Third page of the setup flow - test user credentials, screenshot 08/2023">
+<Frame caption="Third page of the setup flow - insert test user credentials for a log-in test, screenshot 10/2023">
   <img
     src="/images/setup/setup-3-credentials.png"
```
Comment (fabianhinz) [inline first-steps.mdx:53]: Screenshot 2023-10-24 at 13 50 36
 
i don't like the removed text on the FAB ;)

File: first-steps.mdx [hunk 2]
```diff
@@ -75,19 +99,20 @@ Ping us on [Discord](https://discord.gg/3ShnZMKRfA) or [drop us a note](mailto:c
...
 
 You can also record any missing test cases using playwright directly from your terminal. See [recorder](/recorder) for
 more information.
 
-<Frame caption="Home screen button to add test cases using a test recorder, screenshot 09/2023">
+<Frame caption="Home screen button to add test cases using a test recorder, screenshot 10/2023">
   <img
-    src="images/recorder/recorder-1.png"
+    src="images/setup/setup-7-recorder
+    .png"
```
[the remark this answers is not in the document]
Comment (fabianhinz) [reply first-steps.mdx:110]: also: why did you decide for a screenshot of the test cases page?
I don't mind it in general. But we should at least tell the user where to find this btn. It's not on the homescreen/overview/wecome page

**Review by fabianhinz — CHANGES_REQUESTED** [review]

File: first-steps.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ma-zah) [reply first-steps.mdx:110]: Yes, cos the screenshot from the home screen looks meh. This prettier. I'll aded to the description. You found parsing error <3

File: discovery.mdx [hunk 3]
```diff
@@ -13,13 +13,29 @@ Playwright code is generated on the fly just before test execution.
...
+<Frame caption="Free promting window for test case discovery, screenshot 10/2023">
   <img
     src="images/free-prompting-UX.png"
     alt="free prompting window for test case discovery"
   />
 </Frame>
 
+## Chaining your tests
+
+When using the AI agent to discover your tests, you can use a dependency to chain them together. Shorter flows are faster, more specific and more reliable.
```
[the remark this answers is not in the document]
Comment (ma-zah) [reply discovery.mdx:25]: good idea

File: first-steps.mdx [hunk 4]
```diff
@@ -51,21 +66,30 @@ Now we need to sign you up. Please, provide an email, so we can get in touch.
...
-  first test cases within minutes. If the discovery of the sign-in test fails,
-  we will investigate. This might take a little longer.
+  We've triggered test case discovery for the tests you selected right away. You
+  will get your first test cases within minutes. If the discovery of any of the
+  tests fails, we will investigate. This might take a little longer.
 </Info>
 
-We will ask you for the most useful test cases, and we'll create 3 to 8 of them.
-We will notify you once we are done with the test case creation. Or you can try out our **brand-new free prompting feature**!
+We will be adding more out-of-the-box automated test cases as we grow. In the meantime, you can try out our **brand-new free prompting feature** or record test cases with **Playwright recorder**.
```
[the remark this answers is not in the document]
Comment (ma-zah) [reply first-steps.mdx:88]: we'll update it in the next merge if needed. need to move on.

File: first-steps.mdx (hunk 1 above)
Comment (ma-zah) [reply first-steps.mdx:53]: I didn't figure out how to remove it entirely. When you make screenshots it's often in the way of things needed to be seen.

**Review by fabianhinz — APPROVED** [review]

**Pull request merged**